h/views/account_signup.py

```diff
@@ -66,7 +57,8 @@ def post(self):
         signup_service = self.request.find_service(name='user_signup')
         signup_service.signup(username=appstruct['username'],
                               email=appstruct['email'],
-                              password=appstruct['password'])
+                              password=appstruct['password'],
+                              privacy_accepted=datetime.datetime.utcnow())
```

Comment:
Populating the `privacy_accepted` field on the new User is the responsibility of the view (via the `user_signup` service), as the population of that field should only happen from true user interaction, not "magically" whenever we create a new User or something.